demisto_sdk/commands/content_graph/objects/list.py

```diff
@@ -19,13 +19,10 @@ def _upload(
         client: demisto_client,
         marketplace: MarketplaceVersions,
     ) -> None:
-        with TemporaryDirectory("w") as f:
-            dir_path = Path(f)
-            self.dump(dir_path, marketplace=marketplace)
-
+        with self.path.open("r") as _list:
```

Comment:
why is this change? (@dorschw fyi)

Reply:
fixes https://jira-hq.paloaltonetworks.local/browse/CIAC-8354
basically we are trying to send a `Path` object in the api request, leading to the following mentioned in the issue.

uploading a list is broken ATM, this PR fixes it